examples/kubernetes/node_group/main-datasource.tf

```diff
@@ -0,0 +1,6 @@
+data "vkcs_kubernetes_node_group" "my_created_nodegroup" {
```

Comment:
DO NOT USE IN OBJECT NAME my, object type name etc

Reply:
understood